src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSSemanticConventions.cs

```diff
@@ -13,6 +13,16 @@ internal static class AWSSemanticConventions
     public const string AttributeAWSDynamoTableName = "aws.table_name";
     public const string AttributeAWSSQSQueueUrl = "aws.queue_url";
 
+    // AWS Bedrock service attributes not yet defined in semantic conventions
+    public const string AttributeAWSBedrockGuardrailId = "aws.bedrock.guardrail.id";
+    public const string AttributeAWSBedrockAgentId = "aws.bedrock.agent.id";
+    public const string AttributeAWSBedrockKnowledgeBaseId = "aws.bedrock.knowledge_base.id";
+    public const string AttributeAWSBedrockDataSourceId = "aws.bedrock.data_source.id";
```

Comment:
What is the progress of defining attributes on semantic conventions? This PR is waiting 1 month

Reply:
So what is the status? I think it is the last thing before decision about _merging_,

Reply:
They are currently still in the planning phase.